frontend/src/components/tracing/tracing.tsx

```diff
@@ -101,24 +107,24 @@ export const Tracing: React.FC = () => {
 };
 
 const LazyVega = React.lazy(() =>
-  import("react-vega").then((m) => ({ default: m.Vega })),
+  import("react-vega").then((m) => ({ default: m.VegaLite })),
```

Comment:
I used Vega because it seems to convert to vega spec and the console logs warnings if you specify vegaLite.
```.js
The given visualization spec is written in Vega, but mode argument sets Vega-Lite.
```